translations/messages.en.yml

```diff
@@ -78,6 +78,8 @@ page.registration.register_user: Register user
 page.registration.error_response: Error response
 page.registration.random_default_error: random registration error
 page.registration.submit_error: Return authentication failed
+page.registration.form.email.label: Emailaddress
+page.registration.form.email.submit: Submit
```

Comment:
Would `Continue` or another slightly stronger call to action be more appropriate here? Submit is covering the bases here, but another label might be better? I'm no a11y/ux expert, but maybe this can be improved on.

I'm also fine with leaving it as is!

Reply:
Next / Volgende seems better indeed. Thank you for pointing this out.